libs/datasets/manual_filter.py

```diff
@@ -83,7 +83,13 @@ def _partition_by_fields(
 ) -> Tuple[pd.DataFrame, pd.DataFrame]:
     """Partitions a DataFrame with timeseries wide dates into two: fields and others."""
     timeseries.check_timeseries_wide_dates_structure(ts_in)
-    mask_selected_fields = ts_in.index.get_level_values(PdFields.VARIABLE).isin(fields)
+
+    # NOTE(sean): By default we only block non-demographic data. The demographic
+    # data does not reach the website, but is used by partners so we want this to flow through
+    # regardless of block status.
+    mask_selected_fields = ts_in.index.get_level_values(PdFields.VARIABLE).isin(
+        fields
+    ) & ts_in.index.get_level_values(PdFields.DEMOGRAPHIC_BUCKET).isin(["all"])
```

Comment:
there doesn't seem to be a pandas function for comparing index values to one value, (like [`Series.eq()`](https://pandas.pydata.org/docs/reference/api/pandas.Series.eq.html)). This could probably be done with boolean indexing, but I think I prefer this as it keeps the syntax consistent with what's above. 

If we have strong feelings though, I can look for a way to compare the index to one value directly